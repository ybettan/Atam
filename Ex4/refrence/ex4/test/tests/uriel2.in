3410 10OBRORFORBAOR2 1 U