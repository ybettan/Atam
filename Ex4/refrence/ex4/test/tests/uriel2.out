Please enter number of rows:
3
Please enter number of columns:
4
Please enter game duration in seconds and max number of moves:
10 10
Please enter the board:
OBRORFORBAOR

We are all set. Total time: 10 sec
Max number of allowed moves: 10 moves
Press Enter key when you are ready...

Current Board:
* B R *
R F * R
B A * R
Moves:
10 moves left, please enter your move:
2 1 U

Current Board:
* B R *
R X * R
B * * R
Final Moves: (2,1,U)
Well done, you won!
